<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Radar</class>
 <widget class="QMainWindow" name="Radar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Характеристики РЛС</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabBeta">
       <attribute name="title">
        <string>ДН по азимуту</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCustomPlot" name="plotAreaBeta" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="labelNxBeta">
                <property name="text">
                 <string>Количество стобцов</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputNxBeta"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="labelDxBeta">
                <property name="text">
                 <string>Расстояние между столбцами</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputDxBeta"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="labelFBeta">
                <property name="text">
                 <string>Частота, МГц</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputFBeta"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="labelTargetBeta">
                <property name="text">
                 <string>Азимут цели, градус</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputTargetBeta"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QGroupBox" name="groupBoxScaleBeta">
              <property name="title">
               <string>Масштаб</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QRadioButton" name="radioButtonLinBeta">
                   <property name="text">
                    <string>Линейный</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButtonLogBeta">
                   <property name="text">
                    <string>Логарифмический</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonPlotBeta">
              <property name="text">
               <string>Построить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonDefaultBeta">
              <property name="text">
               <string>Параметры по умолчанию</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonSaveBeta">
              <property name="text">
               <string>Сохранить в файл</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabEps">
       <attribute name="title">
        <string>ДН по углу места</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QCustomPlot" name="plotAreaEps" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QLabel" name="labelNyEps">
                <property name="text">
                 <string>Количество строк</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputNyEps"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <widget class="QLabel" name="labelDyEps">
                <property name="text">
                 <string>Расстояние между строками</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputDyEps"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <item>
               <widget class="QLabel" name="labelFEps">
                <property name="text">
                 <string>Частота, МГц</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputFEps"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_18">
              <item>
               <widget class="QLabel" name="labelAngleEps">
                <property name="text">
                 <string>Угол наклона АР, градус</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputAngleEps"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_19">
              <item>
               <widget class="QLabel" name="labelHgcEps">
                <property name="text">
                 <string>Высота подъема Hгц, м</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputHgcEps"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_14">
              <item>
               <widget class="QLabel" name="labelTargetEps">
                <property name="text">
                 <string>Угол места цели, градус</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputTargetEps"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QGroupBox" name="groupBoxScaleEps">
              <property name="title">
               <string>Масштаб</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_15">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_16">
                 <item>
                  <widget class="QRadioButton" name="radioButtonLinEps">
                   <property name="text">
                    <string>Линейный</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButtonLogEps">
                   <property name="text">
                    <string>Логарифмический</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkRefractionEps">
              <property name="text">
               <string>Отражения от земли</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonPlotEps">
              <property name="text">
               <string>Построить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonDefaultEps">
              <property name="text">
               <string>Параметры по умолчанию</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonSaveEps">
              <property name="text">
               <string>Сохранить в файл</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPeleng">
       <attribute name="title">
        <string>Пеленгационная характеристика</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_31">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_21">
          <item>
           <widget class="QCustomPlot" name="plotAreaPeleng" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_22">
              <item>
               <widget class="QLabel" name="labelNyPeleng">
                <property name="text">
                 <string>Количество строк</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputNyPeleng"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_23">
              <item>
               <widget class="QLabel" name="labelDyPeleng">
                <property name="text">
                 <string>Расстояние между строками</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputDyPeleng"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_24">
              <item>
               <widget class="QLabel" name="labelFPeleng">
                <property name="text">
                 <string>Частота, МГц</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputFPeleng"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_25">
              <item>
               <widget class="QLabel" name="labelAnglePeleng">
                <property name="text">
                 <string>Угол наклона АР, градус</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputAnglePeleng"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_26">
              <item>
               <widget class="QLabel" name="labelHgcPeleng">
                <property name="text">
                 <string>Высота подъема Hгц, м</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputHgcPeleng"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_27"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QCheckBox" name="checkRefractionPeleng">
              <property name="text">
               <string>Отражения от земли</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonPlotPeleng">
              <property name="text">
               <string>Построить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonDefaultPeleng">
              <property name="text">
               <string>Параметры по умолчанию</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonSavePeleng">
              <property name="text">
               <string>Сохранить в файл</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabProvodka">
       <attribute name="title">
        <string>Проводка по высоте</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_20">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_28">
          <item>
           <widget class="QCustomPlot" name="plotAreaProvodka" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_29">
              <item>
               <widget class="QLabel" name="labelNyProvodka">
                <property name="text">
                 <string>Количество строк</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputNyProvodka"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_30">
              <item>
               <widget class="QLabel" name="labelDyProvodka">
                <property name="text">
                 <string>Расстояние между строками</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputDyProvodka"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_32">
              <item>
               <widget class="QLabel" name="labelFProvodka">
                <property name="text">
                 <string>Частота, МГц</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputFProvodka"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_33">
              <item>
               <widget class="QLabel" name="labelAngleProvodka">
                <property name="text">
                 <string>Угол наклона АР, градус</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputAngleProvodka"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_34">
              <item>
               <widget class="QLabel" name="labelHgcProvodka">
                <property name="text">
                 <string>Высота подъема Hгц, м</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputHgcProvodka"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_36">
              <item>
               <widget class="QLabel" name="labelH0Provodka">
                <property name="text">
                 <string>Высота полета, км</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inputH0Provodka"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_35"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QCheckBox" name="checkRefractionProvodka">
              <property name="text">
               <string>Отражения от земли</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonPlotProvodka">
              <property name="text">
               <string>Построить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonDefaultProvodka">
              <property name="text">
               <string>Параметры по умолчанию</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonSaveProvodka">
              <property name="text">
               <string>Сохранить в файл</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>826</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;Меню</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
